test/suite/test_prepare_hs01.py

```diff
@@ -101,7 +104,21 @@ def prepare_updates(self, uri, ds, nrows, nsessions, nkeys):
             cursors[j].close()
             sessions[j].close()
 
-    @unittest.skip("Temporarily disabled")
+        # Re-read the original versions of all the data. This ensures reading
+        # original versions from the data store as the prepared updates are
+        # aborted
+        cursor = self.session.open_cursor(uri)
+        self.session.begin_transaction('read_timestamp=' + timestamp_str(2))
+        for i in range(1, nsessions * nkeys):
+            cursor.set_key(ds.key(nrows + i))
+            self.assertEquals(cursor.search(), 0)
+            # Correctness Test - commit_value should be visible
+            self.assertEquals(cursor.get_value(), bigvalue1)
+            # Correctness Test - prepare_value should NOT be visible
+            self.assertNotEquals(cursor.get_value(), bigvalue2)
+        cursor.close()
+        self.session.commit_transaction()
+
```

Comment:
Looks like this new section of the test was copied from a fragment above it?  It would be good to factor it into a function to avoid repeated code.

Reply:
Fixed it by creating a new function to avoid duplication.